src/common/types.ts

```diff
@@ -84,3 +86,33 @@ export enum RecordingStatus {
   Recording = "RECORDING",
   Paused = "PAUSED",
 }
+
+export interface JourneyStartEvent {
+  event: "journey/start";
+  data: {
+    name: string;
+  };
+}
+
+export interface JourneyEndEvent {
+  event: "journey/end";
+  data: {
+    name: string;
+    status: "succeeded" | "failed"; // TBC
```

Comment:
Perhaps we should make this `passed: boolean` if we only have two states.

Reply:
This value comes from Playwright so I think we should just keep it, perhaps removing the comment. This way in the backend JS we can simply set this field to `journey.status` from the event object.